unreleased.md

```diff
@@ -4,13 +4,22 @@
 **Added:**
 - react-native-bpk-component-pagination-dots:
   - Introducing the React Native pagination dots component.
-  
+
 - react-native-bpk-component-alert:
   - Introducing the React Native alert component.
 
 - react-native-bpk-component-carousel:
   - Introducing the React Native carousel component.
 
+- bpk-component-icon: 3.19.3 => 3.20.0
+- bpk-mixins: 17.7.0 => 17.8.0
+- bpk-svgs: 5.15.1 => 5.16.0
+- react-native-bpk-component-icon:
+- react-native-bpk-component-button-link: 2.0.5 => 2.0.6
+- react-native-bpk-component-button: 6.0.7 => 6.0.8
+- react-native-bpk-component-navigation-bar: 2.0.2 => 2.0.3
+  - New icon: `mileage`
```

Comment:
The version numbers here shouldn't appear until it's released (the releaser does it after Lerna determines the next version).